### DISCUSSION TITLE
How to install the latest version dpgen and deepmd-kit off-line?

### DISCUSSION THREAD

**Discussion by JohnWhipples** [deepmodeling/dpgen#790] [Q&A] [answered]
my HPC can not access network, thus I can only intall them off-line. The method by pip or conda can not work. 
1. the off-line packages of dpgen is "dpgen-0.10.2-Linux-x86_64.sh" now, not the latest version v0.10.5, then how to intall it off-line?
2. "dpdata" is not necessary to install alone? the installation of "dpgen" or "deepmd-kit" all has installed "dpdata" at the same time?
3. the installation of off-line "deepmd-kit" and "dpgen" will both install cuda? is it necessary? So there is  no need to source the original cuda environment in my HPC?

thanks.

**Comment by HuangJiameng** [comment]
![image](https://user-images.githubusercontent.com/105633685/178169698-36c0a7bc-9c59-4e0c-a232-58b67a9cf88e.png)
You can download the zip file of the latest version 0.10.5 here
For requirements, see [https://github.com/deepmodeling/dpgen/blob/master/requirements.txt](url)

**Comment by HuangJiameng** [comment, marked as answer]
You can try again https://github.com/deepmodeling/dpgen/releases/download/v0.10.6/dpgen-0.10.6-Linux-x86_64.sh
